sdk/src/cameras/ad-96tof1-ebz/camera_96tof1.cpp

```diff
@@ -50,24 +49,24 @@ struct rangeStruct {
 };
 
 // A map that contains the specific values for each revision
-static const std::map<aditof::Revision, std::array<rangeStruct, 3>>
+static const std::map<std::string, std::array<rangeStruct, 3>>
     RangeValuesForRevision = {
-        {aditof::Revision::RevB,
+        {"RevB",
          {{{"near", 250, 800}, {"medium", 300, 3000}, {"far", 3000, 6000}}}},
-        {aditof::Revision::RevC,
+        {"RevC",
          {{{"near", 250, 800}, {"medium", 300, 4500}, {"far", 3000, 6000}}}}};
 
 static const std::string skCustomMode = "custom";
 
+static const std::vector<std::string> availableControls = {
+    "noise_reduction_threshold", 
+    "ir_gamma_correction",
+    "revision"
+};
+
 Camera96Tof1::Camera96Tof1(std::unique_ptr<aditof::DeviceInterface> device)
-    : m_specifics(std::make_shared<aditof::Camera96Tof1Specifics>(
-          aditof::Camera96Tof1Specifics(this))),
-      m_device(std::move(device)), m_devStarted(false) {
-
-    // initialize range values with the default data for revision C
-    auto cam96tof1Specifics =
-        std::dynamic_pointer_cast<aditof::Camera96Tof1Specifics>(m_specifics);
-    cam96tof1Specifics->setCameraRevision(aditof::Revision::RevC);
+    : m_device(std::move(device)), m_devStarted(false),
```

Comment:
`m_devStarted` is defined in the .h file after the `m_availableControls`, should swap their order in the initializer list, otherwise some compilers might signal a warning here.

Reply:
Done